nixos/modules/security/hidepid.nix

```diff
@@ -33,6 +33,12 @@ in
   };
 
   config = mkIf cfg.enable {
+    assertions =
+     [ { assertion = cfg.procGroup != null -> (config.users.groups."${cfg.procGroup}" or "") != "";
+         message = "the group named at security.hideProcessInformation.procGroup does not exist!";
+       }
+     ];
+
```

Comment:
I have considered changing the option to simply indicating whether a group named "proc" should be created and be made exempt instead of letting the user name a group of their choosing. It's actually what I'd prefer; I felt that the current approach has smaller impact on the rest of the system, however.